keps/0001-kubernetes-enhancement-proposal-process.md

```diff
@@ -175,7 +175,7 @@ Metadata items:
     this SIG will take responsibility for the bulk of those artifacts.
   * Sigs are listed as `sig-abc-def`, where the name matches the
     directory entry in the `kubernetes/community` repo.
-* **participating-sigs** Optional
+* **participating-groups** Optional
   * A list of SIGs that are involved or impacted by this KEP.
```

Comment:
we changed sigs to groups, but the explanatory note below still only refers to sigs.. maybe "A list of SIGs, WGs and other groups..." (is there something other than a SIG/WG?)

Reply:
Ahhh nice catch @kikisdeliveryservice !

I'll update this

Reply:
there are committees too (e.g., code of conduct committee)